docs/prow/prow-architecture.md

```diff
@@ -43,6 +43,10 @@ Prow plugins applied for the Kyma project include:
 - **cat** that checks if there is a new GitHub event for a `/meow` comment on a PR. If it finds it, it adds a cat image to the related PR. For that purpose, it uses the GitHub token available as a Kubernetes Secret.
 - **config-updater** that reads the configuration from `config.yaml`, `plugins.yaml`, and the `jobs` folder, and updates it on the production cluster after the merge to the `master` branch. This plugin is only configured for the `test-infra` repository.
 
+## Prow Addons Controller Manager
+
+The Prow Addons Controller Manager contains all custom controller extensions for Prow infrastructure. Currently it contains only the Notifier controller. Notifier watches all ProwJobs and sends notifications to a given Slack channel. Find detailed documentation [here](../../development/prow-addons-ctrl-manager/README.md)
```

Comment:
maybe make it explicit that it was developed by us, and point to the source code.